db.go

```diff
@@ -807,7 +807,7 @@ func (db *DB) Querier(mint, maxt int64) (Querier, error) {
 
 func rangeForTimestamp(t int64, width int64) (mint, maxt int64) {
 	mint = (t / width) * width
-	return mint, mint + width
+	return mint, mint + width - 1
```

Comment:
let's add comment that we explicitly don't want chunk minT sample to be in maxTime block if it is the case because block is `[minT, maxT)`